server/db/models/product.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    unique: true,
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT
+  },
+  price: {
+    type: Sequelize.DECIMAL(10, 2),
```

Comment:
Follow-up comment from #6, don't store money as a float, as floating point math is inaccurate. Store in terms of cents (e.g. `price_cents`) and only where you display the price (frontend) should you divide by 100 to show the dollar figure.

Reply:
Or… store as a float, but then you have to be very careful to always do math by first multiplying all prices by 100, then dividing by 100 again at the end. Or using an accurate math library, which is very slow as it uses software instead of hardware based arithmetic.